frontend/packages/dev-console/integration-tests/features/getting-started/getting_started_tour_dev_perspective.feature

```diff
@@ -0,0 +1,77 @@
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Monitoring tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Search tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Command Line Terminal
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Help
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears is Final Screen saying 'You are ready to go!'
+   And user can see the button labels as 'Got it' & 'Back to tour'
+   And user can clicks on 'Got it' button
+   Then user is taken to Add view in developer perspective
+
+@regression
+Scenario: Quick tour from help menu
+   Given user is in developer perspective
+   When user opens help menu on top right
+   And user clicks on 'Guided Tour' option
+   And the Getting Started experience is automatically displayed
+   And user can see first screen as 'Welcome to Dev perspective' 
+   And user can see the button labels on the first screen are 'Get Started' & 'Close'
+   And user clicks on Get started 
+   And user can see next screen appears on Perspective switcher
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Monitoring tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Search tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Command Line Terminal
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Help
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears is Final Screen saying 'You are ready to go!'
+   And user can see the button labels as 'Got it' & 'Back to tour'
+   And user can clicks on 'Got it' button
+   Then user is taken to Add view in developer perspective
+
+@regression
+Scenario: Stopping Quick tour in mid of the tour
+   Given user is in developer perspective
+   When user opens help menu on top right
+   And user clicks on 'Guided Tour' option
+   And user can see the first screen as 'Welcome to Dev perspective' 
+   And user clicks on Get started 
+   And user can see next screen appears on Perspective switcher
+   And user can see the button labels as 'Next' & 'Back'
+   And user clicks on Back button
+   And user is taken to Add view in developer perspective
```

Comment:
This regression is wrong. Clicking on the back button takes the user to the previous step not close the tour. When you are on any of the steps except Start and Finish. There is `X` button in the top right corner which can be used to close the tour. Once `X` is clicked a closing out modal will open.